# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/SocketProcessBridgeParent.h
# Commit: d4004105a04a
# Full Hash: d4004105a04ad9a473930acb9c6a05d90cd9f028
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-02-23 04:13:22
# Regressor Bug: 1521879
# File Overlap Count: 1
# Description:
#   Bug 1521879 - Part 1: IPC-based MediaTransport implementation r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D17273
# ==============================================================================

diff -r baecf588657c -r d4004105a04a netwerk/ipc/SocketProcessBridgeParent.h
--- a/netwerk/ipc/SocketProcessBridgeParent.h	Thu Feb 21 23:16:40 2019 +0000
+++ b/netwerk/ipc/SocketProcessBridgeParent.h	Thu Feb 21 20:24:54 2019 +0000
@@ -29,6 +29,10 @@
   void ActorDestroy(ActorDestroyReason aWhy) override;
   void DeferredDestroy();
 
+  dom::PMediaTransportParent* AllocPMediaTransportParent() override;
+  bool DeallocPMediaTransportParent(
+      dom::PMediaTransportParent* aActor) override;
+
  private:
   ~SocketProcessBridgeParent();
 